source/includes/options-conf.yaml

```diff
@@ -707,19 +707,22 @@ description: |
    .. versionadded:: 3.4
 
    .. versionchanged:: 3.6
+      \ 
 
-      - Add support for :term:`zlib` compressor.
+   - Add support for :term:`zlib` compressor.
 
-      - Enabled by default. To disable, set to ``disabled``.
+   - Enabled by default using the :term:`snappy` compressor. To
```

Comment:
sounds like we use the snappy compressor to enable this feature.

maybe:
MongoDB enables snappy network compression for communication, by default.  To disable compression ...